### ISSUE TITLE
Incorrect inline style for katex

### ISSUE THREAD

**Issue by ailectra** [andi23rosca/solid-markdown#29]
Hi,

When using remark-math and rehype-katex, there is a display issue. It shows the image on the right instead of the one on the left:
<img width="250" alt="Sans titre3" src="https://github.com/andi23rosca/solid-markdown/assets/77160706/6968a487-d58d-4bfd-8153-ab1ff5439f40">

The one on the left is displayed using `katex.renderToString()` directly, the one on the right uses `SolidMarkdown`.

The issue seems to be because the style properties are camelCase instead of kebab-case:
```
{
  className: 'mord mathnormal',
  style: { marginRight: '0.07153em' },
  key: 'span---0'
}
```

**Comment by andi23rosca** [comment]
yep, you're right, solid uses kebab case style properties
PRs are welcome, otherwise I'll try to get to fixing it in the weekend

**Comment by ailectra** [comment]
Adding the following code to `childProps` from `MarkdownNode` in `renderer.tsx` seems to fix it:

`if (properties.style) {
	properties.style = Object.entries(properties.style).reduce((acc, [key, value]) => {
		const kebabKey = key.replace(/([a-z0-9])([A-Z])/g, '$1-$2').toLowerCase()
		acc[kebabKey] = value
		return acc
	}, {} as Record<string, string>)
}`

**Comment by ailectra** [comment]
Ok I checked your code. The issue seems to stem from the `parseStyle` function. The style is already in kebab-case but gets turned into camelCase by this function. Removing this function entirely fixes the problem.

**Comment by andi23rosca** [comment]
@ailectra just published `v2.0.1` let me know if it fixes all your isses

**Comment by ailectra** [comment]
Yes it works perfectly now! Thank you
